results.py

```diff
@@ -55,7 +56,6 @@
     "DanskBERT",
     "FollowIR-7B",
     "GritLM-7B",
-    "GritLM-7B-noinstruct",
```

Comment:
What happened to this model?

Reply:
Did we add these back in -why was this resolved?

Reply:
I believe these results were deleted in #18 because they were present in the commit before that

Reply:
Ahh right, yes I did remove that. We should probably figure out a good approach to add results for experiment runs to the leaderboard (added an issue here: https://github.com/embeddings-benchmark/mteb/issues/1211)

Reply:
It was removed due to it conflicting with the results of gritLM with instructions (their model meta makes it seem like they are the same model when loaded from `mteb.load_results(...)`

Reply:
Should we add this one back in? (Preliminary PR https://github.com/embeddings-benchmark/results/pull/28)